grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -177,6 +177,39 @@ def section_compaction(outer_panels):
                         ),
                     ],
                 ),
+
+                panels.timeseries_count(
+                    "Compaction Task L0 Select Level Count",
+                    "Count of L0 compact task.",
```

Comment:
We can make this desc detailed so that we can know its meaning from UI conveniently.